Formula/lib/libxc.rb

```diff
@@ -23,6 +23,8 @@ class Libxc < Formula
   def install
     system "cmake", "-S", ".", "-B", "build",
                     "-DENABLE_FORTRAN=ON",
+                    "-DDISABLE_KXC=OFF",
+                    "-DDISABLE_LXC=OFF",
```

Comment:
can you link some references for these changes? why is this necessary.

Reply:
Hello,

it will be used inside the software 
https://github.com/cp-paw
in future and it changes the compiling-time of this library, only. 

I can imagin that other Software, like the cp-paw, also use this extensions.
In the main branch of this library, the default is to use it (if i understand the installation routine correctly ...)

Best regards,
Axel

Reply:
Make sense, I see the changes in the devel branch as well. Thanks!

related commits:
- https://gitlab.com/libxc/libxc/-/commit/b0ce1d8eaf224c733a00f1cbd02e85d7b85bc5ff
- https://gitlab.com/libxc/libxc/-/commit/a161a63499f882b9256815c218a1f007b0a055b7

---

gonna fresh your PR to satisfy the commit style.